src/hooks/use-render.ts

```diff
@@ -36,7 +36,7 @@ const useRender = () => {
   const viewService = useViewService(pageInfo);
   const rect = useSnapshot({ rect: useRect(), layoutService, viewService, model });
   const { qPivotDataPages, isLoading } = useLoadDataPages({ model, layoutService, viewService, pageInfo });
-  const styleService = useMemo(() => createStyleService(theme, layoutService), [theme.name(), layoutService]);
```

Comment:
It needs to be `theme.name()` because the reference to the `theme` object does not change when a theme is changed.

Reply:
done